components/tekton/tasks/deploy-vse-sync-test.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/categories: Testing
+    tekton.dev/displayName: vse-framework
+    tekton.dev/tags: 'golang, vse-sync-testsuite'
+  name: deploy-vse-sync-test
+spec:
+  description: This task runs the vse-sync-testsuite image.
+  params:
+    - description: |
+        The tag for the image you want to run
+      name: kubeconfig_secret
+      type: string
+    - default: /usr/vse/kubeconfig
+      description: |
+        Path that the kubeconfig will be mounted into
+      name: kubeconfig_secret_dest_path
+      type: string
+    - default: /workspace/reports/vse-sync-testsuite.json
+      description: |
+        Path of test results report
+      name: report_path
+      type: string
+  results:
+    - name: ptp-test-result
+      description: ptp test output result
+  steps:
+    - image: 'quay.io/redhat-partner-solutions/vse-sync-test:0.0.3'
```

Comment:
Since this image is private, we're assuming the kubeconfig's pull secret contains credentials for quay.io and has access to this repo?

Reply:
Yes, and maybe it will become public eventually